devtools/client/framework/target-from-url.js

```diff
@@ -71,7 +74,7 @@ exports.targetFromURL = Task.async(function*(url) {
     // Fetch target for a remote chrome actor
     DebuggerServer.allowChromeProcess = true;
     try {
-      id = parseInt(id);
+      id = parseInt(id, 10);
```

Comment:
I agree some more improvements can be made. I am going to keep the changes contained to this `window` branch for now for easier merging with m-c. I reverted the `parseInt` change outside the window path.